elastic/kibana
# [Fleet] Expose agent logging level in agent policy settings

Closes https://github.com/elastic/kibana/issues/158861

## Summary
Expose agent logging level in agent policy settings. 
The new setting is added via the new settings framework and will show up under "advanced settings". It's currently hidden until the agent supports it.

### Testing
Enable the settings config: https://github.com/elastic/kibana/pull/180597#discussion_r1562448034
- Go to the agent policy settings form
- Under advanced settings there is a new dropdown "Agent logging level". Choose a value and save the policy
- The new value should be retained after saving
- Go to the agent policies tab and select action "View policy"
- The new field should be visible under `agent.logging.level`

<details>
  <summary> Screenshots</summary>
  
![Screenshot 2024-04-12 at 16 21 46](https://github.com/elastic/kibana/assets/16084106/b3083bb5-703a-44c6-a00d-da9d64a2b083)
![Screenshot 2024-04-12 at 16 21 52](https://github.com/elastic/kibana/assets/16084106/bd934262-86f0-4b11-b7b4-d4be72f00715)
![Screenshot 2024-04-12 at 16 22 28](https://github.com/elastic/kibana/assets/16084106/4a352c82-f274-4779-9718-85135f84b0c8)
![Screenshot 2024-04-12 at 16 27 48](https://github.com/elastic/kibana/assets/16084106/11e20051-f91b-44c1-b795-76ef6804cf78)

</details>

### Checklist
